src/main/java/com/hms_networks/americas/sc/thingworx/utils/HttpUtils.java

```diff
@@ -5,7 +5,6 @@
 import com.hms_networks.americas.sc.extensions.fileutils.FileAccessManager;
 import com.hms_networks.americas.sc.extensions.logging.Logger;
 import com.hms_networks.americas.sc.thingworx.TWConnectorConsts;
-
```

Comment:
formatting change in non-formatting commit

Reply:
Can we all decide right now to allow formatting changes in non-formatting commits?

Reply:
This change does not fit with the scope of the commit. Our goal with version control should be a well documented history of the reason for every line change. Using a git blame on this section of code would reveal that it was removed because of "Resolved Javadoc warning" which doesnt make any sense. I understand how frustrating it is to have to go back and fix such a trivial issue.

Reply:
I think its best to maintain the current trend of formatting commits because it ensures that the scope of commits are always (or should always be) exactly what is described in the commit title/body

Reply:
All my commits contain formatting changes. It's the last thing I do before I commit.